store/prefs.js

```diff
@@ -414,4 +419,37 @@ export const actions = {
 
     return dispatch('set', { key: THEME, value });
   },
+
+  setBrandStyle({ rootState, rootGetters }, dark = false) {
+    if (rootState.managementReady) {
+      try {
+        const brandSetting = rootGetters['management/byId'](MANAGEMENT.SETTING, 'brand');
+
+        if (brandSetting && brandSetting.value && brandSetting.value !== '') {
+          const brand = brandSetting.value;
```

Comment:
@nwmac @mantis-toboggan-md Do we _need_ this for 2.6?  The idea of having alternate styles for rancher vs suse is fine, but I just do not like anything about this custom theme implementation and would like to just drop it.

- I have to get a file into assets/brand at build-time somehow; how does this help a user actually apply their own theme at runtime?
- It's in an uncanny-valley that allows users to override only a small part of the theme.  Most customers are either fine with the theme as it is, or are going to want to customize more than just one color.  E.g. some color choices will require changing other colors for text on top of it to be readable; the "what contrasts more" function is not really a good solution for that.
- It's done in JS, which means it doesn't work in SSR.  You see the regular theme and then the brand value seconds later after rehydration is done.
- It's duplicating all sorts of chunks of the SCSS rules and functions with more JS that is brittle and has to be kept in sync
- It's breaking things if used that aren't broken with the hasStylesheet=true version, like the login page: 
![image](https://user-images.githubusercontent.com/753917/116487824-ba7e9600-a845-11eb-8942-957d244a574c.png)